.github/workflows/build.yml

```diff
@@ -28,12 +28,17 @@ jobs:
       BUILD_PLATFORM: x64
 
     steps:
-    - uses: actions/checkout@629c2de402a417ea7690ca6ce3f33229e27606a5
+    - name: Harden Runner
+      uses: step-security/harden-runner@9b0655f430fba8c7001d4e38f8d4306db5c6e0ab # v1
+      with:
+        egress-policy: audit # TODO: change to 'egress-policy: block' after couple of runs
+
+    - uses: actions/checkout@b0e28b5ac45a892f91e7d036f8200cf5ed489415 # v3
       with:
         submodules: 'recursive'
 
     - name: Add MSBuild to PATH
-      uses: microsoft/setup-msbuild@becb80cf9a036187bb1e07e74eb64e25850d757a
+      uses: microsoft/setup-msbuild@becb80cf9a036187bb1e07e74eb64e25850d757a # v1
```

Comment:
I don't recommend keeping this `# v1` comments because dependabot doesn't update them. They almost immediately go out of date.

Reply:
> I don't recommend keeping this `# v1` comments because dependabot doesn't update them. They almost immediately go out of date.

Thanks for the info! I will update the tool to not add these comments.

Reply:
Removed from this PR